msal/src/androidTest/java/com/microsoft/identity/client/PublicClientApplicationTest.java

```diff
@@ -323,69 +323,70 @@ String getFinalAuthUrl() throws UnsupportedEncodingException {
-            void mockHttpRequest() throws IOException {
-                final HttpURLConnection mockedConnection = AndroidTestMockUtil.getMockedConnectionWithFailureResponse(
-                        HttpURLConnection.HTTP_BAD_REQUEST, AndroidTestUtil.getErrorResponseMessage("invalid_request"));
-                Mockito.when(mockedConnection.getOutputStream()).thenReturn(Mockito.mock(OutputStream.class));
-                HttpUrlConnectionFactory.addMockedConnection(mockedConnection);
-            }
-
-            @Override
-            void makeAcquireTokenCall(final PublicClientApplication publicClientApplication,
-                                      final Activity activity,
-                                      final CountDownLatch releaseLock) {
-
-
-                publicClientApplication.acquireToken(activity, SCOPE, "somehint", new AuthenticationCallback() {
-                    @Override
-                    public void onSuccess(IAuthenticationResult authenticationResult) {
-                        fail();
-                    }
-
-                    @Override
-                    public void onError(MsalException exception) {
-                        assertTrue(exception instanceof MsalServiceException);
-
-                        final MsalServiceException serviceException = (MsalServiceException) exception;
-                        assertTrue(MsalServiceException.INVALID_REQUEST.equals(serviceException.getErrorCode()));
-                        assertTrue(!serviceException.getMessage().isEmpty());
-                        assertTrue(serviceException.getHttpStatusCode() == HttpURLConnection.HTTP_BAD_REQUEST);
-                        releaseLock.countDown();
-                    }
-
-                    @Override
-                    public void onCancel() {
-                        fail();
-                    }
-                });
-            }
 
-            @Override
-            String getFinalAuthUrl() throws UnsupportedEncodingException {
-                return mRedirectUri + "?code=1234&state=" + AndroidTestUtil.encodeProtocolState(
-                        ALTERNATE_AUTHORITY, new HashSet<>(Arrays.asList(SCOPE)));
-            }
-
-            @Override
-            protected void performAdditionalVerify(Activity testActivity) {
-                Mockito.verify(testActivity).startActivityForResult(Mockito.argThat(
-                        new ArgumentMatcher<Intent>() {
-                            @Override
-                            public boolean matches(Intent argument) {
-                                final String data = argument.getStringExtra(Constants.REQUEST_URL_KEY);
-                                return data.startsWith(ALTERNATE_AUTHORITY);
-                            }
-                        }), Matchers.eq(RequestCodes.LOCAL_AUTHORIZATION_REQUEST));
-            }
-        }.performTest();
+        // TODO: Revisit this once we're ready to update test cases for ISingle and IMuliple PCA.
```

Comment:
Given that this test was previously annotated as `@Ignore` and has now been totally nixed for compilation issues, I'm willing to say we can just delete rather than carry around the dead code.

You could consider leaving the test name/stub, removing the code and marking as `// TODO` - I have no issue with that